app/controllers/bibliographic_controller.rb

```diff
@@ -295,17 +295,37 @@ def bib_id_param
       sanitize(params[:bib_id])
     end
 
+    def needs_locator_call_no(location)
+      # For now only Firestore items need this.
+      return true if (location || "").start_with?("firestone$")
+      false
+    end
+
+    def make_sortable_call_number(call_no)
+      if call_no.start_with?("Oversize ")
```

Comment:
There was some discussion that "oversize" isn't the only possible value here, for example we may have Folio or other values. In the ticket there's some discussion of checking solr or doing an API call. Otherwise we may need to see if we can configure a more exhaustive list? Given that these items are only going to be in firestone stacks maybe generating a list of possible values here would work fine; not sure how many special designated areas there might be.

Reply:
We created https://github.com/pulibrary/bibdata/blob/5c1f6ed3f4b2a20bb98c8e72ddbbdf3519cf257e/marc_to_solr/lib/traject_config.rb#L1321 for this purpose. It does not include $k.